mule-extensions-api/src/main/java/org/mule/runtime/extension/api/util/NameUtils.java

```diff
@@ -288,12 +288,11 @@ public static String defaultNamespace(String extensionName) {
     if (isBlank(extensionName)) {
       throw new IllegalArgumentException("No Namespace can be created from an empty name");
     }
-
-    String namespace = deleteWhitespace(extensionName);
+    String namespace = extensionName.replace(" ", "-");
```

Comment:
you should trim() before making this replacement.

Reply:
Done.